tmva/tmva/inc/TMVA/DNN/RMSProp.h

```diff
@@ -0,0 +1,247 @@
+// @(#)root/tmva/tmva/dnn:$Id$
+// Author: Ravi Kiran S
+
+/**********************************************************************************
+ * Project: TMVA - a Root-integrated toolkit for multivariate data analysis       *
+ * Package: TMVA                                                                  *
+ * Class  : TRMSProp                                                              *
+ * Web    : http://tmva.sourceforge.net                                           *
+ *                                                                                *
+ * Description:                                                                   *
+ *      RMSProp Optimizer Class                                                   *
+ *                                                                                *
+ * Authors (alphabetical):                                                        *
+ *      Ravi Kiran S      <sravikiran0606@gmail.com>  - CERN, Switzerland         *
+ *                                                                                *
+ * Copyright (c) 2005-2018:                                                       *
+ *      CERN, Switzerland                                                         *
```

Comment:
The copy-right statement is not correct (nobody is from Uni Bonn).

Reply:
I am not aware of the copy-right statement. Everytime, I am copying it from some other existing file and changing the author name alone.

Reply:
Ok I'll remove the U. of Bonn.